packages/offchain-resolver-backend/src/http/profile.ts

```diff
@@ -6,6 +6,7 @@ import {
 import { ethers } from 'ethers';
 import express from 'express';
 import { WithLocals } from './types';
+import { SiweMessage } from 'siwe';
 
 //The test msg should just be the sg of an ethereum address
 const MSG_START = 0;
```

Comment:
I think we can get rid of this since we're using the siwe class to parse the message